### DISCUSSION TITLE
Register non-input values

### DISCUSSION THREAD

**Discussion by van-vince** [react-hook-form/react-hook-form#10094] [Q&A] [answered]
I have this part of my invoice table which the values come from calculation from input values of other part of the table. i am finding it difficult to send this values to the backend..cos i think its not registered properly.. I need help how to register values which are not input values
![Screenshot 2023-03-12 at 11 29 45 PM](https://user-images.githubusercontent.com/67305320/224580854-733adf87-891f-4255-a2cf-d6e7ef8f2418.png)

**Comment by leapful** [comment, marked as answer]
@van-vince 

You can register the field with `<input type="hidden" {...register('discount')} value={ccyFormat(0)} />` so the input is not shown on UI but its value will submitted along with the form as normal.

**Reply by van-vince** [reply]
The values need to be shown on the UI

**Reply by leapful** [reply]
@van-vince 

It seems you still mis-understand the solution, `<input type="hidden" />` will be used along with your `<TableCell>`, you don't need to hide the values.

For example:

```
<TableCell>
   <input type="hidden" {...register('discount')} value={ccyFormat(0)} />
   {ccyFormat(0)}
</TableCell>
```

**Reply by van-vince** [reply]
ok i get it now
